projects/addon-mobile/components/sheet-dialog/sheet-dialog.component.ts

```diff
@@ -54,6 +60,11 @@ export class TuiSheetDialogComponent<I> implements AfterViewInit {
         },
     };
 
+    stuck$ = this.scroll$.pipe(
```

Comment:
How about we make it a behavior subject and next to it the actual boolean only if it changes? This way we get rid of the middle man.

Reply:
done, take a look plz if I got you right